marianas_process_loop_umi_1.8.0/marianas_process_loop_umi.cwl

```diff
@@ -0,0 +1,68 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: marianas_process_loop_umi_cwl
+baseCommand:
+  - java
+inputs:
+  - id: fastq1
+    type: File?
+    inputBinding:
+      position: 1
+  - id: fastq2
+    type: File?
+    inputBinding:
+      position: 2
+  - id: umi_length
+    type: int?
+    inputBinding:
+      position: 3
+outputs:
+  - id: processed_fastq_1
+    type: File
+    outputBinding:
+      glob: '$(inputs.fastq1.basename.replace(''.fastq'', ''_umi-clipped.fastq''))'
```

Comment:
wouldn't `.gz` be the extension here

Reply:
`$(inputs.fastq1.basename.replace(''.fastq.gz'', ''_umi-clipped.fastq.gz''))`